dev-util/pycharm-community/pycharm-community-2024.3.2.ebuild

```diff
@@ -0,0 +1,231 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop readme.gentoo-r1 wrapper xdg-utils
+
+DESCRIPTION="Intelligent Python IDE with unique code assistance and analysis"
+
+# NOTE upstream release info
+# https://data.services.jetbrains.com/products?code=PCP,PCC&release.type=release
+# https://data.services.jetbrains.com/products?code=PCP,PCC&release.type=release&fields=name,releases
+
+HOMEPAGE="https://www.jetbrains.com/pycharm/"
+SRC_URI="
+	amd64? (
+		https://download.jetbrains.com/python/${P}.tar.gz
+	)
+	arm64? (
+		https://download.jetbrains.com/python/${P}-aarch64.tar.gz
+	)
+	x86? (
+		https://download.jetbrains.com/python/${P}.tar.gz
+	)
+"
+
+if [[ "${PN}" == *-professional ]]; then
+	S="${WORKDIR}/${PN/%-professional/}-${PV}"
+fi
+
+LICENSE="|| ( JetBrains-business JetBrains-classroom JetBrains-educational JetBrains-individual )
+	Apache-2.0
+	BSD
+	BSD-2
+	CC0-1.0
+	CC-BY-2.5
+	CC-BY-3.0
+	CPL-1.0
+	CDDL-1.1
+	EPL-1.0
+	GPL-2
+	GPL-2-with-classpath-exception
+	ISC
+	JDOM
+	JSON
+	LGPL-2+
+	LGPL-2.1
+	LGPL-3
+	MIT
+	MPL-1.1
+	MPL-2.0
+	OFL-1.1
+	UPL-1.0
+	ZLIB
+"
+
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="+bundled-jdk"
+
+if [[ "${PN}" == *-professional ]]; then
+	IUSE+=" +bundled-xvfb"
+fi
+
+BDEPEND="
+	dev-util/debugedit
+	sys-apps/file
+	sys-apps/findutils
+	sys-apps/grep
```

Comment:
file, findutils, grep are part of `@system` set, not needed in `BDEPEND`

Reply:
> file, findutils, grep are part of `@system` set, not needed in `BDEPEND`

I was hoping for this. Thank you!